src/shared/helpers/variantHelpers.ts

```diff
@@ -14,16 +14,26 @@
  * limitations under the License.
  */
 
-import { ValueSetMetadata } from '../thoas/metadata';
+export const getReferenceAndAltAlleles = <
+  T extends {
+    allele_type: {
+      value: string;
+    };
+  }
+>(
+  alleles: Array<T>
+) => {
+  const referenceAlleleIndex = alleles.findIndex(
+    (allele) => allele.allele_type.value === 'biological_region'
+  );
 
-export type VariantStudyPopulation = {
-  name: string;
-  size: number;
-  description: string;
-  type: ValueSetMetadata;
-  is_global: boolean;
-  is_from_genotypes: boolean;
-  display_group_name: string; // The name of the group of populations this population belongs to; serves for grouping purposes
-  super_population: VariantStudyPopulation | null;
-  sub_populations: VariantStudyPopulation[];
+  const referenceAllele = alleles[referenceAlleleIndex];
+  const alternativeAlleles = alleles.filter(
```

Comment:
Can you delete the reference allele from the alleles array in the step above to avoid this?

Reply:
Why would that be any better?